docs/google/engine.md

```diff
@@ -0,0 +1,103 @@
+# Quantum Engine
+
+The Quantum Engine, via the `cirq.google.Engine` class, executes programs and jobs using the
+Quantum Engine API. 
+
+Note that the Quantum Engine API is not yet open for public access.
+
+## Authenticating to Google Cloud
+
+Before you begin, you will need to create a Google Cloud project with the API
+enabled and billing enabled.  You will then to create credentials in order to
+access the API.
+
+You can create application default credentials from the command line using the
+gcloud client:
+
+`gcloud auth application-default login`
+
+From a colab, you can execute:
+
+```
+from google.colab import auth
+auth.authenticate_user(clear_output=False)
+```
+
+More information on creating application default credentials can be found on the
+[Google cloud](https://cloud.google.com/docs/authentication/production) website.
+
+## Engine class
+
+The `Engine` class is the entry point to communicate with the API.
+
+It can be initialized using your project id (found within your
+[Google Cloud Platform Console](https://console.cloud.google.com)).
+You can use this instance to run quantum circuits or sweeps (parameterized
+variants of a general circuit).
+
+```
```

Comment:
Tag as python. You may also want to tag this block as one that shouldn't execute when testing the documentation.

I'm surprised that this doesn't cause the doc tests to fail, actually, since it should attempt to connect and fail.

Reply:
Done.  It seems that the doctest only tests cirq/**.py right now.

Reply:
Nm, was looking in the wrong place.  docs/snippets_test.py is the thing that tests this.
Fixed the test by adding appropriate substitution to use a simulator.